account_financial_report_webkit/report/templates/aged_open_invoices.mako

```diff
@@ -158,19 +158,21 @@
                             %endfor
                         </div>
                     </div>
-                %endfor  # end of the loop on partners
-                <div class="act_as_row labels">
-                    <div class="act_as_cell total account_title bg">${_('Total')}</div>
-                    <div class="act_as_cell"></div>
-                    <div class="act_as_cell"></div>
-                    <div class="act_as_cell"></div>
-                    <div class="act_as_cell"></div>
-                    <div class="act_as_cell"></div>
-                    <div class="act_as_cell amount classif total account_title bg">${formatLang(aged_open_inv[account.id]['balance']) | amount}</div>
-                    %for classif in ranges:
-                        <div class="act_as_cell amount classif total account_title bg">${formatLang(aged_open_inv[account.id][classif]) | amount }</div>
-                    %endfor
                 </div>
+                %endfor  # end of the loop on partners
+                <div class="act_as_table list_table" style="margin-top:5px;">
```

Comment:
Needed to create a new table, and had to give the cells the same width as the original thead.

Reply:
Although I've never messed with `mako` templates. Isn't it bad practice to use explicit `px` values for styling? Is it possible to replace this with percentages or apply a class to the header and the cells to give them the proper width?

Reply:
If i'm right, a pdf has always the same width, and i wanted to keep up with the headers: https://github.com/EssentNovaTeam/account-financial-reporting/pull/9/files/0cf68ed7ea6f96734c5bcd503722dae0310389d3#diff-14e11f3e8bf645ac73de13680b415c379ee9f3428f4f8afa158f1a8f5c98812cR104-R116 ; otherwise the diff would be much bigger if i have to rewrite that part too.

Reply:
Looks to me like fixed widths are used throughout preexisting code indeed.